Quadriga/pom.xml

```diff
@@ -31,14 +31,14 @@
 		<quadriga.admin.password>admin</quadriga.admin.password>
 		<pullrequest></pullrequest>
 		<textfiles.location></textfiles.location>
-		<transformationfiles.location></transformationfiles.location>
-		<email.host></email.host>
-		<email.port></email.port>
-		<email.username></email.username>
-		<email.pw></email.pw>
-		<email.from></email.from>
+		<transformationfiles.location>/Users/yoganandakishore/transformationFiles</transformationfiles.location>
+		<email.host>abcd@email.com</email.host>
+		<email.port>25</email.port>
+		<email.username>abcd</email.username>
+		<email.pw>efgh</email.pw>
+		<email.from>lkjh@gmail.com</email.from>
```

Comment:
those should go into a your maven settings file rather than here.
https://diging.atlassian.net/wiki/display/CHC/How+to+override+Maven+properties